### PR TITLE
feat: add cardano

### PR DESCRIPTION
**Pull request by detherminal**

### CODE REVIEW

File: lib/wallets/api/cardano/blockfrost_http_provider.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import '../../../utilities/logger.dart';
+
+class BlockfrostHttpProvider implements BlockfrostServiceProvider {
+  BlockfrostHttpProvider(
+      {required this.url,
+        this.version = "v0",
+        this.projectId,
+        http.Client? client,
+        this.defaultRequestTimeout = const Duration(seconds: 30)})
```
Comment (sneurlax) [inline lib/wallets/api/cardano/blockfrost_http_provider.dart:14]: We should add an optional proxyInfo parameter here and use it if not null.

**Comment by sneurlax** [comment]
I just need to doublecheck tor support / usage, but otherwise this looks good to me.

**Comment by detherminal** [comment]
Tor support has been resolved by commit [1ec7ee95d268e5f9bff29d04a77fee5a0f9ebb72](https://github.com/cypherstack/stack_wallet/pull/970/commits/1ec7ee95d268e5f9bff29d04a77fee5a0f9ebb72) @sneurlax @julian-CStack

**Review by sneurlax — DISMISSED** [review]

**Review by sneurlax — APPROVED** [review]
latest commit since last review is good

**Comment by sneurlax** [comment]
Included in / superceded by #995

**Pull request closed without merging**